zoloto/marker.py

```diff
@@ -60,7 +62,7 @@ def spherical(self):
 
     @property
     def cartesian(self):
-        return ThreeDCoordinates(*self._tvec)
+        return ThreeDCoordinates(x=self._tvec[0], y=self._tvec[1], z=self._tvec[2],)
```

Comment:
I don't believe this change is necessary?

Reply:
98ab842